src/_data/main-nav.yml

```diff
@@ -190,6 +190,9 @@
     - label: Checkout
       url: /howdoi/checkout/checkout_overview.html
 
+    - label: Compliance
+      url: ../../../compliance/industry-compliance.html
```

Comment:
```suggestion
      url: /compliance/industry-compliance.html
      versionless: true
```

Reply:
Initially I had the paths the way you suggest, but kept getting a 404. The only way I could get it to work was to bump up a few levels. I'll try again.

Reply:
Fixed.

Reply:
@ccondra, you didn't fix this. It's much easier if you  use the GitHub interface to accept these suggestions. Just click **Commit suggestion**.

Reply:
Also, in your local branch, you removed the forward slash at the beginning of the `url:` value. That will cause link errors. You must start the file path with a forward slash.